src/app/zap-templates/zcl/data-model/silabs/ha.xml

```diff
@@ -2068,14 +2068,18 @@ limitations under the License.
         This message is used to return household appliance current alerts.
       </description>
       <arg name="alertsCount" type="AlertCount"/>
+      <!--
       <arg name="alertStructures" type="AlertStructure" array="true"/>
+      -->
```

Comment:
Please at the very least document why things are commented out.  Possibly TODO comments...

Reply:
Done, thanks